checks_windows.go

```diff
@@ -0,0 +1,410 @@
+package main
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	wapi "github.com/iamacarpet/go-win64api"
+	"golang.org/x/sys/windows/registry"
+)
+
+func (c cond) BitlockerEnabled() (bool, error) {
+	const FULLY_ENCRYPTED = 1
+	const ENCRYPTION_IN_PROGRESS = 2
```

Comment:
why do we have 2 constants here -- is it not possible to do `if status.ConversionStatus == 1 || status.ConversionStatus == 2` ?
I get that it makes the code slightly more readable, but we are adding two lines for almost no benefit

Reply:
This is existing code and those are the official status codes from Microsoft documentation. We could update the dependency because I created a PR to add these constants